content/participants/ttjc@poojakelkar/index.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Pooja Kelker
+type: "participant"
+name: Pooja Kelkar"
+
```

Comment:
Hey! @poojakelkar don't remove the **portfolio** from here. If you dont have one right now you can just leave it blank like:
* portfolio: ""